pkgs/development/python-modules/bbox/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyquaternion
+}:
+
+buildPythonPackage rec {
+  pname = "bbox";
+  version = "0.9.2";
+
```

See https://github.com/varunagrawal/bbox/blob/master/pyproject.toml#L9

```suggestion
  version = "0.9.2";

  disabled = pythonOlder "3.6";
```